lib/DDGC/Schema/Result/User.pm

```diff
@@ -4,6 +4,8 @@ package DDGC::Schema::Result::User;
 
 use Moo;
 extends 'DDGC::Schema::Result';
+with 'DDGC::Schema::Role::Result::User::Subscription';
+
```

Comment:
I also dislike long classes.  

What I did once was pull out all the functions for these Result classes into Roles.  I decided that code and schema definition were 2 separate things and belonged in two separate files.  I kind of liked that.  So there was one role for a corresponding Result class and the role was definitely not reusable.  

I'm a strong believer that doing something just for readability is enough of a justification to do it.  I make lots of small new functions that are only used once.